src/ui/profile_preferences.ui

```diff
@@ -991,6 +991,71 @@
             </property>
            </widget>
           </item>
+          <item row="3" column="0" colspan="2">
+           <widget class="QWidget" name="widget_timerDebugOutputMinimumInterval" native="true">
+            <layout class="QHBoxLayout" name="horizontalLayout_timerDebugOutputMinimumInterval" stretch="0,1">
+             <property name="leftMargin">
+              <number>0</number>
+             </property>
+             <property name="topMargin">
+              <number>0</number>
+             </property>
+             <property name="rightMargin">
+              <number>0</number>
+             </property>
+             <property name="bottomMargin">
+              <number>0</number>
+             </property>
+             <item>
+              <widget class="QLabel" name="label_timerDebugOutputMinimumInterval">
+               <property name="sizePolicy">
+                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
+                 <horstretch>1</horstretch>
+                 <verstretch>0</verstretch>
+                </sizepolicy>
+               </property>
+               <property name="text">
+                <string>Show debug info for functional Timers only when their interval is equal to or more than (h:mm:ss.zzz):</string>
```

Comment:
I don't think this is very intuitive to understand - how about:

> Don't show timers in Debug Console less than:

Reply:
Yes, but it is still too lengthy.

Reply:
The shortest I've now got it to is:

Show *Central Debug Console* messages for working *Timers* with an interval at least (h:mm:ss.zzz)

Is "*Central Debug Console*" the term we are using for that `TConsole` instance in user-land though? I see one reference to "Debug window" in the Wiki [documentation for GMCP](https://wiki.mudlet.org/w/Manual:Supported_Protocols#GMCP) but strangely I cannot find any other places in the Wiki where it is mentioned...!

Reply:
I'd just call it "debug window", that's what people see and know it is as.